### PR TITLE
add .history folder to gitignore

### PR DESCRIPTION
**Pull request by samuelstein**
Ignore local history changes from .history folder

### CODE REVIEW

**Comment by samuelstein** [comment]
sure: https://dailydotnettips.com/2015/05/04/how-to-navigate-through-history-of-files-in-visual-studio-code/. It's a relatively new feature of vs code.

File: Global/VisualStudioCode.gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+.history
```
Comment (shiftkey) [inline Global/VisualStudioCode.gitignore]: Shouldn't this be inside `.vscode/`? If so, it should be covered by the rule on line 1

Comment (samuelstein) [reply Global/VisualStudioCode.gitignore]: no the folder is outside of .vscode

Comment (shiftkey) [reply Global/VisualStudioCode.gitignore]: I'd like to see some documentation that explains this file better - describing the feature is one thing, but it'd be nice to confirm we're looking at the right file in the right location before merging this in.

It also seems suboptimal for this to not be using the `.vscode` folder, especially if it's an in-the-box feature of VSCode, for what looks like a generated file.

**Comment by sara0871** [comment]
👋

**Comment by samuelstein** [comment]
i could not find better official documentation. @shiftkey try it for yourself. got to user settings and activate it via `"local-history.enabled": 1,`

**Comment by shiftkey** [comment]
On VSCode 1.22.2 I don't seem to be able to enable this feature:

**Comment by shiftkey** [comment]
Closing this out as I'm not able to understand the requested change and verify it's correct

**Pull request closed without merging**